olmo/model.py

```diff
@@ -473,20 +474,42 @@ def reset_parameters(self):
             self.k_norm.reset_parameters()
         if self.q_norm is not None:
             self.q_norm.reset_parameters()
-        init_weights(
-            self.config,
-            self.attn_out,
-            d=self.config.d_model,
-            layer_id=self.layer_id,
-            type_of_module=ModuleType.out_module,
-        )
-        init_weights(
-            self.config,
-            self.ff_out,
-            d=self.ff_out.in_features,
-            layer_id=self.layer_id,
-            type_of_module=ModuleType.out_module,
-        )
+
```

Comment:
nit: we call `init_normal` in every block here on the same modules, so I think it would be cleaner to structure this in the same way as you've structured it in `OLMoSequentialBlock`, where you just set the `std` and `cutoff_factor` in the `if-elif` blocks.

Reply:
Yes, I planned to do it after we cleared up the confusion regarding the extra multiplier.